web/src/misc/UaHelper.ts

```diff
@@ -5,3 +5,17 @@ export function processUserAgent(userAgent: string) {
   const isMobile = !!md.mobile() && !md.tablet();
   return { userAgent, isMobile };
 }
+
+export function isMobileFromReq(req: any) {
```

Comment:
Can't we type the req ?

Reply:
Not sure if NextJS has typings for `req`